tycho-p2/tycho-p2-repository-plugin/src/main/java/org/eclipse/tycho/plugins/p2/repository/MavenP2SiteMojo.java

```diff
@@ -200,6 +204,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                 if (file == null || !file.isFile()) {
                     continue;
                 }
+                if (isSkippedDeploy(mavenProject)){
```

Comment:
Now we are adding more (and maybe more in the future) conditions and checks I think it would be good to extract the whole block into a method:

```
if (includeReactor) {
            resolveReactorProjects(mavenProject, bundles, advices);
}
```

Reply:
fixed